iym/admin.py

```diff
@@ -0,0 +1,342 @@
+from django.contrib import admin
+from iym import models
+from io import StringIO
+import petl as etl
+from decimal import Decimal
+from urllib.parse import urlencode
+from slugify import slugify
+from zipfile import ZipFile
+from django.conf import settings
+from django_q.tasks import async_task
+
+import os
+import csv
+import tempfile
+import requests
+import hashlib
+import base64
+import json
+import time
+import re
+import iym
+import datetime
+
+RE_END_YEAR = re.compile(r"/\d+")
+
+MEASURES = [
```

Comment:
let's move the importing code to a dedicated module so that admin.py can focus on being admin

Reply:
at least in a tasks.py which is a common place to put async tasks code, but even better would be for the code actually doing the upload work to be separate from the task coordinating the upload and feeding that information back to the user in the upload object.

Reply:
done